velox/vector/ComplexVector.h

```diff
@@ -109,6 +109,10 @@ class RowVector : public BaseVector {
     return childrenSize_;
   }
 
+  // Resize a row vector by adding trailing nulls to the top level row without
+  // resizing children.
```

Comment:
Would you add that the caller must ensure that vector is singly-referenced and nulls are mutable?

Would you add a test for this new API?

Reply:
nit: also mention that it can only increase the size